src/stories/roboto_condensed.stories.mdx

```diff
@@ -7,8 +7,14 @@ import { alphabetTable, numberTable, sampleTexts } from './helpers'
   .roboto-condensed {
     font-family: 'Roboto Condensed';
   }
+  .bold {
+    font-weight: 700;
+  }
 </style>
-<div class="roboto-condensed has-text-black">
+<div class="roboto-condensed has-text-black has-bottom-margin">
+  ${storyFn()}
+</div>
+<div class="source-sans-pro bold has-text-black ">
```

Comment:
@dhruvkb Sorry for this will update all the instances
Also can you please help as if should I update all the font files for Roboto Condensed in the `fonts` and `src/fonts` both